pkg/vex/csaf.go

```diff
@@ -0,0 +1,81 @@
+package vex
+
+import (
+	csaf "github.com/csaf-poc/csaf_distribution/v3/csaf"
```

Comment:
What if use `https://github.com/openvex/go-vex/blob/main/pkg/csaf/csaf.go`

Reply:
I'd rather use the one maintained by OASIS CSAF TC, see https://oasis-open.github.io/csaf-documentation/tools.html